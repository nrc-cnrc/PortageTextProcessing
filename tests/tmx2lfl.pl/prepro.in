Here is a literal soft hyphen: ­ followed by random text.
 	  This line    has superfluous whitewspace   	
||| Testing the three pipes ||| like so.  How about two sets ||| ||| like this? |||
And how about a BOM?﻿
And lots of control characters: ⁠ and trailing text.
Literal \x2011 a‑b \xAD a­b \x1E ab \x1F ab.
