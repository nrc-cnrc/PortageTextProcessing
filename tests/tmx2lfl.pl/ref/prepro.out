Here is a literal soft hyphen: - followed by random text.
This line has superfluous whitewspace
___|||___ Testing the three pipes ___|||___ like so. How about two sets ___|||___ ___|||___ like this? ___|||___
And how about a BOM?
And lots of control characters: and trailing text.
Literal \x2011 a-b \xAD a-b \x1E a-b \x1F ab.
